mrp_bom_component_menu/views/mrp_bom_component_view.xml

```diff
@@ -8,11 +8,11 @@
         <field name="name">mrp.bom.line.tree.view</field>
         <field name="model">mrp.bom.line</field>
         <field name="arch" type="xml">
-            <tree string="Components">
-                <field name="bom_id" />
-                <field name="product_id"/>
-                <field name="product_qty"/>
-                <field name="routing_id"/>
+            <tree string="Components" create="0">
+                <field name="bom_id" readonly="1" />
+                <field name="product_id" readonly="1" />
+                <field name="product_qty" readonly="1" />
+                <field name="routing_id" readonly="1" />
```

Comment:
Does this actually solve the issue? I think the form still opens unless you set an editable attribute to the tree tag.